support-frontend/app/views/newContributions.scala.html

```diff
@@ -159,7 +159,7 @@ <h1 class="gu-content__blurb-header">Help us deliver the independent journalism
         </div>
       </div>
 
-      <div class="gu-content__bg"><svg class="svg-contributions-bg-desktop" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1400 620"><circle fill="#E7D4B9" cx="213" cy="273" r="70"></circle><circle fill="#FEC8D3" cx="1280" cy="369" r="140"></circle><circle fill="#BB3B7F" cx="1152" cy="136" r="125"></circle><circle fill="#00B2FF" cx="138" cy="475" r="145"></circle><circle fill="#FF4E36" cx="748" cy="165" r="58"></circle><circle fill="#FAB375" cx="915" cy="395" r="225"></circle><circle fill="#FFFFFF" cx="1169" cy="549" r="70"></circle></svg></div></main><footer class="component-footer" role="contentinfo"><div class="component-footer__content"><small class="component-footer__copyright">
+      </main><footer class="component-footer" role="contentinfo"><div class="component-footer__content"><small class="component-footer__copyright">
```

Comment:
this is the old circles SVG. it was getting in my way when messing with the background - pretty sure it can go